apps/openmw/mwclass/npc.cpp

```diff
@@ -863,16 +864,27 @@ namespace MWClass
             return action;
         }
 
-        if(getCreatureStats(ptr).isDead())
-            return std::shared_ptr<MWWorld::Action>(new MWWorld::ActionOpen(ptr, true));
-        if(ptr.getClass().getCreatureStats(ptr).getAiSequence().isInCombat())
-            return std::shared_ptr<MWWorld::Action>(new MWWorld::FailedAction("#{sActorInCombat}"));
-        if(getCreatureStats(actor).getStance(MWMechanics::CreatureStats::Stance_Sneak)
```

Comment:
I don't see this branch handled in the new code, could be why this commit breaks pickpocketing.